### ISSUE TITLE
Library Updating

### ISSUE THREAD

**Issue by JOJOyesyesyes** [tachiyomiorg/tachiyomi#6677]
### Steps to reproduce

I only get to update 10 mangas x comics when i got 120+ in my library now i need to manually update each alone
![Screenshot_20220218_234418_eu kanade tachiyomi](https://user-images.githubusercontent.com/78759906/154772433-fc288291-5b9b-425d-8991-45b9e97ba618.jpg)
![Screenshot_20220218_234427_eu kanade tachiyomi](https://user-images.githubusercontent.com/78759906/154772439-50b09614-96ba-4ca2-b694-b8e3c9b93544.jpg)

### Expected behavior

fix this issue

### Actual behavior

I only get to update 10 mangas x comics when i got 120+ in my library now i need to manually update each alone

### Crash logs

no crashes thankfully

### Tachiyomi version

0.13.1

### Android version

10

### Device

Huawei honor 10

### Other details

this issue started only after updating the app yesterday

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by arkon** [comment]
#6599
